elytron/ELY-2320-integrity-for-FilesystemSecurityRealm.adoc

```diff
@@ -0,0 +1,152 @@
+= [ELY-2320] Add integrity support to FilesystemSecurityRealm
+:author:            Ashpan Raskar
+:email:             araskar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This objective of this proposal is to add the ability to verify integrity for the filesystem realm, this will be done with the use of a Public and Private Key pair, that will create a signature on every identity on the xml identity file.
```

Comment:
> This objective of this proposal is to add the ability to verify integrity for the filesystem realm

The whole filesystem realm, or just individual identities? I'm able to add a new identity with name of my choice just by copying an existing one. It is also possible to override an existing identity this way.

Reply:
I've changed it so now you could copy and existing identity but any other change to the identity (eg changing attributes, name, credentials) will flag as an error next time the identity is being referenced